### PR TITLE
feat(client & server): Production Page

### PR DESCRIPTION
**Pull request by SiuYe**
#

## Changes this PR introduces

- Production Page UI
- Production End Point
- Production Controller Unit Test

## How to test this PR

- Test End Points
- Make sure the production page renders
- Make sure the production displays values

https://share.clickup.com/t/h/nq08fz/WNC6I6C01LP5JNB

## Notes for reviewers

- 🥂

## Checklist

- [x] Have I renamed the title of this pull request to follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)?
- [ ] I will squash and merge when my PR is approved.
- [x] I have added tests to cover my changes (if applicable).

### CODE REVIEW

**Comment by Mackbellemore** [comment]
Task linked: [CU-nq08fz Production page](https://app.clickup.com/t/nq08fz)

File: server/app/controllers/ProductionController.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      const production = await this.productionService.deleteProduction(request.params.id);
+      return this.json(production);
+    } catch (error) {
+      return this.handleError(error);
+    }
+  }
+
+  @httpPatch('/:id')
+  public async update(request: Request): Promise<results.JsonResult> {
+    console.log('yes?');
```
Comment (Mackbellemore) [inline server/app/controllers/ProductionController.ts:63]: yes man

**Review by gordonpn — CHANGES_REQUESTED** [review]
it's broken

File: server/app/controllers/ProductionController.ts (hunk 1 above)
Comment (SiuYe) [reply server/app/controllers/ProductionController.ts:63]: LOL! whoops...

**Comment by Mackbellemore** [comment]
> it's broken

its because the backend and frontend are same PR @gordonpn

**Comment by gordonpn** [comment]
> > it's broken
> 
> 
> 
> its because the backend and frontend are same PR @gordonpn 

don't tell me what to do

File: client/src/components/ProductionPage/Productions.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            <Th>Expected Start Date</Th>
+            <Th>Expected End Date</Th>
+            <Th>Actual Start Date</Th>
+            <Th>Actual End Date</Th>
+            <Th>Manufacture</Th>
+            <Th>Note</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {data.data
```
Comment (DavidLiang01) [inline client/src/components/ProductionPage/Productions.jsx:67]: put `isSuccess `condition here to avoid undefined data value

**Pull request closed without merging**